frontend/shared/components/src/periods/EditRegistrationPeriodView.vue

```diff
@@ -60,12 +67,38 @@ const pop = usePop();
 
 const { patched, addPatch, hasChanges, patch } = usePatch(props.period);
 
+useValidation(errors.validator, validate);
+
+function validate() {
+    if (patched.value.endDate.getTime() < patched.value.startDate.getTime()) {
+        const message = $t('De einddatum moet na de startdatum liggen.');
+
+        // prevent the error from flickering
+        if (errors.errorBox?.errors.message !== message) {
```

Comment:
De flicker is er om duidelijk te maken dat het opnieuw is gecheckt

Reply:
deze code lijkt me heel erg foutgevoelig - als je zoiets moet oplossen moet dat op hogere niveau's in plaats van in elke mogelijke validation (want dat probleem zal in alle andere views evengoed zijn)

Reply:
(zou het ook gewoon niet oplossen omdat het zo werkt als bedoelt)